recipe/meta.yaml

```diff
@@ -31,8 +31,6 @@ requirements:
     - {{ cdt('xorg-x11-proto-devel') }}
     - libxcb
     - xcb-proto
-    - xorg-xcb-proto
-    - xorg-libxcb
```

Comment:
I think these are needed.

Reply:
These have been archived as they're duplicates of the un-prefixed (`xorg-`) packages. See conda-forge/xorg-xcb-proto-feedstock#2 and conda-forge/xorg-libxcb-feedstock#2.

Reply:
These have been archived as they're duplicates of the un-prefixed (xorg-) packages. See conda-forge/xorg-xcb-proto-feedstock#2 and conda-forge/xorg-libxcb-feedstock#2.

Reply:
Oh I see. Thanks